meetings/notes-2018-04-20.md

```diff
@@ -54,7 +54,7 @@ Agenda and notes
         * **Conclusion** - We’ll switch the spec to copy
     4. [PartitionNumberPattern treats -0 as 0](https://github.com/tc39/ecma402/issues/219) (Jack Horton)
         * Shane - we always handled it differently
-            * (Steven: ICU history: [http://bugs.icu-project.org/trac/ticket/8333#comment:14](http://bugs.icu-project.org/trac/ticket/8333#comment:14) [http://bugs.icu-project.org/trac/ticket/13551](http://bugs.icu-project.org/trac/ticket/13551) )
+            * (Steven: ICU history: [https://unicode-org.atlassian.net/browse/ICU-8333#comment:14](https://unicode-org.atlassian.net/browse/ICU-8333#comment:14) [https://unicode-org.atlassian.net/browse/ICU-13551](https://unicode-org.atlassian.net/browse/ICU-13551) )
```

Comment:
https://unicode-org.atlassian.net/browse/ICU-8333?focusedCommentId=46005#comment-46005

Reply:
thanks, this is now fixed!